internal/cli/inputs.go

```diff
@@ -32,8 +34,10 @@ func (i ProjectInputs) Filter() realm.AppFilter { return realm.AppFilter{i.Proje
 
 // Flags registers the project app input flags to the provided flag set
 func (i *ProjectInputs) Flags(fs *pflag.FlagSet) {
+	fs.StringVarP(&i.App, flagApp, flagAppShort, "", flagAppUsage)
+
 	fs.StringVar(&i.Project, flagProject, "", flagProjectUsage)
-	fs.StringVar(&i.App, flagApp, "", flagAppUsage)
+	flags.MarkHidden(fs, flagProject)
```

Comment:
[q] Is there a reason the project flag is to be hidden? I imagine there would be a use case where someone would want to see apps filtered by project

Reply:
historically that project flag was really only useful for mongo employees (who have access to _a lot_ of apps).  in a similar vein to the base urls for realm/atlas, I made them hidden.